pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractTopic.java

```diff
@@ -157,10 +161,14 @@ public abstract class AbstractTopic implements Topic, TopicPolicyListener<TopicP
 
     protected final LongAdder msgOutFromRemovedSubscriptions = new LongAdder();
     protected final LongAdder bytesOutFromRemovedSubscriptions = new LongAdder();
+    protected final LongAdder bytesOutFromRemovedSystemSubscriptions = new LongAdder();
+    protected final LongAdder msgOutFromRemovedSystemSubscriptions = new LongAdder();
```

Comment:
The PIP not contains any metric like `pulsar_broker_out_msgs_total_internal`, please avoid this change